pkg/osquery/Dockerfile

```diff
@@ -48,6 +48,7 @@ USER root
 #collect all files and prepare them into a tar file
 RUN mkdir -p /opt/osquery/lenses /opt/osquery/yara /opt/osquery/extensions \
  && cp /home/"$OSQUERY_BUILD_USER"/osquery/build/osquery/osqueryd /opt/osquery \
```

Comment:
I know this is a separate issue altogether. However, osqueryd needs to be copied as **/opt/osquery/hubble_osqueryd**. Refer [here](https://github.com/hubblestack/hubble/blob/develop/hubblestack/extmods/modules/nebula_osquery.py#L1291)